packages/grafana-ui/src/components/Alert/InlineBanner.story.tsx

```diff
@@ -45,37 +42,22 @@ Basic.args = {
   severity: 'error',
   title: 'Basic',
 };
-Basic.parameters = {
-    control: { exclude: ['buttonContent', 'withButton']},
-};
 
 export const WithActions: ComponentStory<typeof Alert> = (args) => {
   return (
-    <Alert {...args} buttonContent={args.withButton ? args.buttonContent : undefined} onRemove={action('Remove button clicked')}>
-        <VerticalGroup>
-          <div>Child content that includes some alert details, like maybe what actually happened.</div>
-        </VerticalGroup>
-      </Alert>
+    <Alert {...args}>
+      <VerticalGroup>
+        <div>Child content that includes some alert details, like maybe what actually happened.</div>
+      </VerticalGroup>
+    </Alert>
   );
 };
 
-WithActions.argTypes = {
-    withButton: { 
-        control: { type: 'select', options: [true, false] } 
-    },
-}
-
 WithActions.args = {
   title: 'With action',
   severity: 'error',
   onRemove: action('Remove button clicked'),
   buttonContent: 'Close',
-  withButton: true
 };
 
-WithActions.parameters = {
-  controls: {
-    exclude: ['onRemove'],
```

Comment:
Why did we delete this since we can't do anything to this control? (same in Toast)

Reply:
That's a fair point, even though it is not how we are doing it in other places for now... I think at the moment we tell people to use the "props" table in documentation to see all the props that exist with their description?